MITADS-Speech/corpora_importer.py

```diff
@@ -177,22 +181,32 @@ def _maybe_convert_wav(self,mp3_filename, wav_filename):
             except sox.core.SoxError:
                 pass
 
+    ##overrider this to filter
+    def row_validation(self,filename,duration,comments):
+        ##return True 
+        return True
+
     def one_sample(self,sample):
-        mp3_wav_filename = sample
-        # Storing wav files next to the mp3 ones - just with a different suffix
+
+        mp3_wav_filename = sample[0]
+        make_wav_resample = sample[1]
+        # Storing wav files next to the audio filename ones - just with a different suffix
         wav_filename = path.splitext(mp3_wav_filename)[0] + ".wav"
-        self._maybe_convert_wav(mp3_wav_filename, wav_filename)            
+
+        ##Note: to get frames/duration for mp3/wav audio we not use soxi command but sox.file_info.duration(
+        ##soxi command is not present in Windows sox distribution  - see this  https://github.com/rabitt/pysox/pull/74
+        duration = sox.file_info.duration(mp3_wav_filename)
+        comments = sox.file_info.comments(mp3_wav_filename)
```

Comment:
pySox does not handle char encoding, so it raise an error when trying to decode bytes to utf8
```suggestion
        comments = ""
        try:
          comments=sox.file_info.comments(mp3_wav_filename)
        except UnicodeError as e:
          try:
            completedProcess=subprocess.run(["soxi", "-a", mp3_wav_filename], stdout=subprocess.PIPE)
            comments=completedProcess.stdout.decode("utf-8", "ignore")
          except:
            pass
```